demos/common/cpp/models/src/detection_model_yolo.cpp

```diff
@@ -67,11 +70,16 @@ void ModelYolo3::prepareInputsOutputs(InferenceEngine::CNNNetwork& cnnNetwork) {
             auto outputLayer = outputInfo.find(op->get_friendly_name());
             if (outputLayer != outputInfo.end()) {
                 auto regionYolo = std::dynamic_pointer_cast<ngraph::op::RegionYolo>(op);
+
                 if (!regionYolo) {
                     throw std::runtime_error("Invalid output type: " +
                         std::string(op->get_type_info().name) + ". RegionYolo expected");
                 }
 
+                if(!regionYolo->get_mask().size()) {
+                    isYoloV3 = false;
+                }
```

Comment:
```suggestion
                isYoloV3 = regionYolo->get_mask().size() ? true : false
```
Would it work here?

Reply:
current version will work even if we have several outputs. New one - not. Not sure if it would be applicable in future, but let's keep current version.